test/e2e-test/testdata/app/app_delay_suspend.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: delay-suspend-test
+spec:
+  components:
+    - name: delay-suspend-comp
+      type: webservice
+      properties:
+        image: nginx
+        port: 80
+  policies:
+    - name: local-deploy
+      type: topology
+      properties:
+        clusters: [ "local" ]
+        namespace: default
+  workflow:
+    steps:
+      - name: suspend-test
+        type: suspend
+        properties:
+          waitDuration: 30s
+      - name: deploy-to-local
```

Comment:
We can delete the policy and just use `apply-component` here.

Reply:
ok, already modified.